### PR TITLE
Fix for issue #30379

### PR DESCRIPTION
**Pull request by isuruRuhu**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/30379

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: misc/ballerinalang-data-mapper/src/main/java/org/ballerinalang/datamapper/AIDataMapperCodeActionUtil.java [hunk 1]
```diff
@@ -139,6 +139,12 @@ public static AIDataMapperCodeActionUtil getInstance() {
         }
 
         List<DiagnosticProperty<?>> props = diagnostic.properties();
+
+        //To check if the left is a symbol (Fix issue 30379)
```
Comment (nadeeshaan) [inline misc/ballerinalang-data-mapper/src/main/java/org/ballerinalang/datamapper/AIDataMapperCodeActionUtil.java:143]: Do we need to mention the issue number here? We can have a test case instead.

**Review by nadeeshaan — APPROVED** [review]

**Review by nadeeshaan — CHANGES_REQUESTED** [review]
https://github.com/ballerina-platform/ballerina-lang/pull/30706/files#r636636005

File: misc/ballerinalang-data-mapper/src/main/java/org/ballerinalang/datamapper/AIDataMapperCodeActionUtil.java (hunk 1 above)
Comment (isuruRuhu) [reply misc/ballerinalang-data-mapper/src/main/java/org/ballerinalang/datamapper/AIDataMapperCodeActionUtil.java]: Removed the issue numb and added a test case.

**Review by nadeeshaan — APPROVED** [review]
LGTM

**Review by nadeeshaan — APPROVED** [review]
LGTM

**Review by malithjayasinghe — APPROVED** [review]

**Pull request merged**